### PR TITLE
Removes buggy datetime until we have a more robust solution

### PR DESCRIPTION
**Pull request by andycarrell**

### CODE REVIEW

**Review by theramis — APPROVED** [review]

File: app/pages/index.tsx [hunk 1]
```diff
@@ -104,10 +101,6 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
                 Ministry of Health NZ
               </a>
             </p>
-            <div className="hidden sm:block">&#8226;</div>
-            <p title={formatTimeTitle(dataFetchedAtTimeUtc)}>
```
Comment (jishaal) [inline app/pages/index.tsx:108]: Is the bug with `dataFetchedAtTimeUtc` being incorrect intermittently?

Comment (andycarrell) [reply app/pages/index.tsx:108]: The bug is "updated less than an hour ago" never changes, for example if you leave the browser tab open for longer than an hour

Comment (jishaal) [reply app/pages/index.tsx:108]: Oh right! A solution would be to have a  component that takes in the date, run in a useEffect with setInterval to update every hour to solve the staleness problem

**Review by jishaal — APPROVED** [review]

File: app/pages/index.tsx (hunk 1 above)
Comment (andycarrell) [reply app/pages/index.tsx:108]: Yeh that sounds great actually 🕺

**Pull request merged**